test/karma-setup.js

```diff
@@ -221,6 +222,12 @@ function resetDefaultOptions(testName) {
 
     Highcharts.setOptions(defaultOptionsRaw);
 
+    // Restore radial axis defaults
+    Highcharts._modules['Core/Axis/RadialAxis.js'].RadialDefaultOptions =
+        Highcharts.merge(
+            Highcharts._modules['Core/Axis/RadialAxisDefaults.js']
+        );
```

Comment:
This will break when we go webpack and `Highcharts._modules` no longer exists. Instead, I would expose `Highcharts.RadialAxis` (from `masters/highcharts-more.src.ts`). The karma-setup can copy those properties before the tests start ([like we do with defaultOptions](https://github.com/highcharts/highcharts/blob/cd5be8300aa6f6c6c1e74cc4efcb13cb0f1b48ad/test/karma-setup.js#L130)), then restore it here.